config/application.rb

```diff
@@ -47,9 +47,14 @@ class Application < Rails::Application
         load override
       end
     end
-  end
 
```

Comment:
settings changes are high risk.  This worries me; need to check the raw file rather than the diff view.

Reply:
This seems ok -- have we been running around with an extra `end` in this file?